installing_on_kubernetes/_topics/installing_vddk.md

```diff
@@ -0,0 +1,64 @@
+## Installing the VMware-vix-disklib (VDDK) on containers
+
+To install the VDDK on containers we can take an existing container and create another layer on top of it.  Once built, this image will need to be pushed to a registry and the CR updated to reflect that.
+
+1. Fetch the VDDK linux tar file from VMware. (`VMware-vix-disklib-[version].x86_64.tar.gz`)
+    **Note:**
+
+      - If you do not already have a login ID to VMware, then you will
+        need to create one. At the time of this writing, the file can be
+        found by navigating to menu:Downloads\[vSphere\]. Select the
+        version from the drop-down list, then click the **Drivers &
+        Tools** tab. Expand **Automation Tools and SDKs**, and click
+        **Go to Downloads** next to the **VMware vSphere Virtual Disk
+        Development Kit** version. Alternatively, find the file by
+        searching for it using the **Search** on the VMware site.
+
+      - See VMware documentation for information about their policy
+        concerning backward and forward compatibility for VDDK.
+
+2. Set up a temporary directory where we will build the image
+```bash
+$ mkdir -p /tmp/vddk_container/container-assets
+```
+
+3. Copy the tar file into the container-assets directory.  For Example:
+```bash
+$ cp ~/Downloads/VMware-vix-disklib-7.0.1-16860560.x86_64.tar.gz /tmp/vddk_container/container-assets
```

Comment:
Should this be a generic version value and we point to another document describing what versions of vddk we support and against which versions of vsphere?  cc @agrare

Reply:
Done